tests/Unit/EmogrifierTest.php

```diff
@@ -1461,20 +1458,7 @@ public function addAllowedMediaTypeKeepsStylesForTheGivenMediaType()
     /**
      * @test
      */
-    public function emogrifyAddsMissingHeadElement()
-    {
-        $this->subject->setHtml('<html></html>');
-        $this->subject->setCss('@media all { html { some-property: value; } }');
-
-        $result = $this->subject->emogrify();
-
-        static::assertContains('<head>', $result);
-    }
```

Comment:
Shouldn't this be retained as `Emogrifier` doesn't extend `AbstractHtmlProcessor`?